tests/ReflectionTarget.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Brick\Reflection\Tests;
+
+/**
+ * The Reflection Target class.
+ */
+class ReflectionTarget
+{
+    /**
+     * @param string $foo
```

Comment:
Should be `@var`

Reply:
Ah, just noticed that you're actually testing that it returns no types. Anyway, I don't think it's a good idea to put an invalid docblock in here. Maybe the best thing to do is to remove the docblock altogether?

Reply:
I tihnk it's not the good way to remove the $foo docblock bcause it will output the following error message.

```
1) Brick\Reflection\Tests\ReflectionToolsTest::testGetPropertyTypesShouldReturnEmptyArray
TypeError: preg_match() expects parameter 2 to be string, boolean given

/home/lee/reflection/src/ReflectionTools.php:174
/home/lee/reflection/tests/ReflectionToolsTest.php:56

```
It will be failed when excuting the ```preg_match``` function.

Reply:
This was a bug in ReflectionTools, I never realized that `getDocComment()` could return false! I just fixed it, and [reported the documentation mistake](https://bugs.php.net/bug.php?id=76060) on php.net.

You can now safely remove the docblock in the test!